assets/js/base/hooks/shipping/use-shipping-rates.js

```diff
@@ -28,6 +35,8 @@ import { pluckAddress } from '../../utils';
  */
 export const useShippingRates = ( addressFieldsKeys ) => {
 	const { cartErrors, shippingRates } = useStoreCart();
+	const { addErrorNotice } = useStoreNotices();
+	const throwError = useThrowError();
 	const addressFields = Object.fromEntries(
```

Comment:
@nerrad when solving the conflicts in this PR, notice `Object.fromEntries()` was causing issues in IE11 (see #1941). Just to let you know so we don't re-introduce the issue. :slightly_smiling_face:

Reply:
just noting, I forgot that with the changes in another [pull](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1960), this logic isn't even needed now.